src/index.html

```diff
@@ -11,7 +11,151 @@
+          How old are You?
+          <input
+            type="number"
+            name="UserAge"
+            value="12"
+            min="1"
+            max="100"
+          >
+        </label>
+        <label class="form">
+          Full date of birth:
+          <input
+            type="date"
+            name="DateOfBirth"
+            placeholder="mm/dd/yyyy"
+          >
+        </label>
+        <label class="form">
+          I accept the term of the agreement
+          <input
+            type="checkbox"
+            name="checkbox"
+            value="ok"
+            required
+          >
+        </label>
+      </fieldset>
+
+      <fieldset class="block">
+        <legend>Registration</legend>
+        <label class="form">
+          E-mail:
+          <input
+            type="email"
+            name="UserEmail"
+            placeholder="email@example.com"
+            required
+          >
+        </label>
+        <label class="form">
+          Password:
+          <input
+            type="password"
+            name="password"
+            required
+          >
+        </label>
+      </fieldset>
+
+      <fieldset class="block">
+        <legend>An interesting fact about you!</legend>
+        <div class="form">
+          Do you love cats?
+          <label>
+            <input
+              type="radio"
+              name="LoveCats"
+              value="y"
+            >
+            Yes
+          </label>
+          <label>
+            <input
+              type="radio"
+              name="LoveCats"
+              value="n"
+            >
+            No
+          </label>
+        </div>
+        <label class="form">
+          What is your favorite color?
+          <input type="color" name="color">
+        </label>
+        <label class="form">
+          What time do you go to bed?
+          <input
+            type="time"
+            name="time"
+            step="1"
+          >
+        </label>
+        <label class="form">
+          What are your favorite brand of cars?
+          <select name="select" multiple>
+            <option value="BMW">BMW</option>
+            <option value="Audi">Audi</option>
+            <option value="Lada">Lada</option>
+          </select>
+        </label>
+        <label class="form">
+          How do you rate our work?
+          <input
+            type="range"
+            name="RateWork"
+            value="0"
+          >
+        </label>
+      </fieldset>
+      <fieldset class="block">
+        <legend>Additional info:</legend>
+        <label class="form">
+          Comments:
+          <textarea
+              name="comments"
+              cols="20"
+              rows="2"
+              minlength="10"
+              maxlength="50"
+            >
+          </textarea>
+        </label>
+        <label class="form">
+          Would you recommend us?
+          <select name="select">
+            <option value="Yes" selected>yes</option>
```

Comment:
all text in attributes should start with lowercase letter

```suggestion
            <option value="yes" selected>yes</option>
```

Reply:
Thank you 😊